R/utils.r

```diff
@@ -4,8 +4,26 @@ from_json <- function(obj) {
   simplify_homogeneous_lists(jsonlite::fromJSON(obj, simplifyVector = FALSE))
 }
 
+has_names <- function(obj) {
+  !is.null(names(obj)) && !is.null(Find(function(s) !is.na(s) && (s != ""), names(obj)))
```

Comment:
I don't believe it's possible for names to be all blank in practice, so `!is.null(names(obj))` should be sufficient.